topics_publish:
    state: "/state"
    visualization: "/visualization"
topics_subscribe:
    orderId: "/orderId"
    orderUpdateId: "/orderUpdateId"
    zoneSetId: "/zoneSetId"
    lastNodeId: "/lastNodeId"
    lastNodeSequenceId: "/lastNodeSequenceId"
    nodeStates: "/nodeStates"
    edgeStates: "/edgeStates"
    agvPosition: "/agvPosition"
    velocity: "/agvVelocity"
    loads: "/loads"
    driving: "/driving"
    paused: "/paused"
    newBaseRequest: "/newBaseRequest"
    distanceSinceLastNode: "/distanceSinceLastNode"
    actionStates: "/actionStates"
    batteryState: "/batteryState"
    operatingMode: "/operatingMode"
    errors: "/errors"
    information: "/information"
    safetyState: "/safetyState"
